src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -94,6 +97,26 @@ class K8sServiceImpl implements K8sService {
     @Nullable
     private String requestsMemory
 
+    @Value('${wave.build.k8s.toleration.arm64.enabled:false}')
```

Comment:
Note that tolerations aren't tied to arm64 only, so hardcoding it as under the arm64 section is restrictive.

Moreover, customers may need to add multiple different tolerations to the wave build pods, so we should define a way to add multiple tolerations.

Please read the related [taint and toleration docs](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).

Reply:
Ok, but we're hardcoding a single specific toleration use case, but customers may need to install wave in other environments and may need other (even multiple) tolerations, not just ARM vs AMD nodes.
Think about customer environments where they have GPU nodes, or newer generation CPUs, and they may want to only allow wave build pods in a specific location.
Let me know if this helps.

Reply:
You are correct in that part, but in this case, I am checking whether the platform you asked for is AMR and then applying this toleration. 

https://github.com/seqeralabs/wave/blob/d9c1bfd939c6abbb7622e02fb5dce70aa94a948d/src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy#L406-L409

We can provide another set of properties for a general tolerations, which will be applied to all build pods

Reply:
Then why hardcoding our special ARM usecase? Let's build a generic toleration system and we'll use that in our use case

Reply:
because we only want ARM pods to run on nodes with arm taint and not AMD 
or am I missing something?

Reply:
That's correct.
My point is: why making a special usecase to add tolerations for ARM only and then a another generic system to add other tolerations? That's why I was proposing to create the generic tolerations system only, and get rid of the special ARM usecase.

Reply:
In a quick call we were able to share our points: we need to have tolerations for each architecture that wave supports (amd64 and arm64), and users can pass multiple tolerations, so the multiple string parameters will need to change to become a single `List<Toleration>` object for each arch.